### ISSUE TITLE
Parameter Type Serializer Escaping Issue 

### ISSUE THREAD

**Issue by tempura-san** [sabre-io/vobject#24]
Hi everybody,

I am facing some problems with ownCloud 4.5.7. My aim is to synchronise my N900 with ownCloud. On the N900 I am using syncevolution as synchronisation engine.

After doing a two-way sync all specific information is lost. For example telephone numbers are stripped of the information if they are cell, work, and so on. This will only happen during a two way sync, as on a one-way import the data is imported and displayed in ownCloud correctly. Problem is getting it back to the device.

So I came up with CURL just to see what is exported by ownCloud. For TEL it is exported as:

```
TEL;TYPE=HOME\,VOICE:...
```

But it should be:

```
TEL;TYPE=HOME,VOICE:...
```

By escaping comma the value list "HOME", "VOICE" is made a single value "HOME,VOICE". As an effect syncevolution refuses to parse the unknown value and omits/dumps the information.

For reference the relevant sections of RFC2426 (as ownCloud exports VCARD v3):

```
2.5 Structured Type Values

...

   Lists of values are delimited by a list delimiter, specified by the
   COMMA character (ASCII decimal 44). A COMMA character in a value MUST
   be escaped with a BACKSLASH character (ASCII decimal 92).

```

and

```

3.4.  Property Value Escaping

   Some properties may contain one or more values delimited by a COMMA
   character (U+002C).  Therefore, a COMMA character in a value MUST be
   escaped with a BACKSLASH character (U+005C), even for properties that
   don't allow multiple instances (for consistency).

...

   Furthermore, some fields of compound properties may contain a list of
   values delimited by a COMMA character.  Therefore, a COMMA character
   in one of a field's values MUST be escaped with a BACKSLASH
   character, even for fields that don't allow multiple values (for
   consistency).  Compound properties allowing multiple instances MUST
   NOT be encoded in a single content line.

...

   In all other cases, escaping MUST NOT be used.
```

So basically escaping ',' should not be used for elements on the left side of the colon.
I have not yet seen an example of "a COMMA character in a value".

Relevant code section is located in 3rdparty/Sabre/VObject/Parameter.php:66.
Disabling the comma escaping solves the issue for, but I am not quite sure about the side effects.

Please let me know your opinions on this as I am no expert on this topic.

Cheers,
Tempura
